docs-site/docs/autograd.md

```diff
@@ -148,3 +148,978 @@ for epoch in range(epochs):
         for p, gp in autograd.backward(loss):  # auto backward
             sgd.update(p, gp)
 ```
+
+### Supported operators
+
+#### mean
+
+Init a element-wise Mean operator.
+
+##### Input
```

Comment:
remove '####'?
when I view this file on github, the `mean` and `Input` are of the same size. Not easy to read.

Reply:
I add a `---` to separate them, it'd be better now.